CHANGES/5495.misc

```diff
@@ -0,0 +1 @@
+Add publish support for comps.xml
```

Comment:
Please add a separate entry for 5495.feature to have user-facing changelog which should mention sync and publish of comps.xml types.